examples/systemjs/src/app.component.ts

```diff
@@ -15,12 +15,14 @@ import {TranslateService} from 'ng2-translate/ng2-translate';
     </div>
   `,
 })
-export class AppComponent {
-    constructor(private translate: TranslateService) {
-        translate.addLangs(["en", "fr"]);
-        translate.setDefaultLang('en');
+export class AppComponent implements OnInit {
+    constructor(private translate: TranslateService) { }
+  
+    ngOnInit() {
```

Comment:
Used `ngOnInit` because it's best practice to keep the constructor as light as possible.